frame/staking/src/pallet/impls.rs

```diff
@@ -1634,9 +1634,9 @@ impl<T: Config> Pallet<T> {
 		ensure!(
 			T::VoterList::iter()
 				.all(|x| <Nominators<T>>::contains_key(&x) || <Validators<T>>::contains_key(&x)),
-			"VoterList contains non-nominators"
+			"VoterList contains non-staker"
 		);
-		T::VoterList::try_state()?;
```

Comment:
Because in most cases is a different pallet, which will have its own opportunity of calling its `try_state`. 

Note that this is not a good approach, as we cannot know this (`VoterList` is a new pallet) in all cases. For now I removed it because the `try_state` of staking was already taking too long.